Documentation/DocuBlocks/Rest/Administration/JSF_get_admin_server_id.md

```diff
@@ -6,12 +6,16 @@
 
 @RESTDESCRIPTION
 
```

Comment:
please remove the empty line here, this triggers the duplication bug the user noticed.